### PR TITLE
Restyles slab, horizontal cards, and icon combinations on Reg/Reporting

### PR DESCRIPTION
**Pull request by jenniferthibault**
## Summary

These changes bring the registration and reporting slabs closer into pattern alignment with the recent implementation on the [Advanced Data page](https://beta.fec.gov/data/advanced/)

1. I'm finding clarity in the icon styling system: creating a "simple" one-color style for anything that lives in a regular card. The color of the icons is determined by the background color pair of the card. This adds new icons for:
 - `training`
 - `info_person`
 - `calendar`
 - and renames the former `calendar` to `grid` 

  screen shot 2016-06-14 at 2 47 34 am

2. Adds neutral slab to the style guide, officially
screen shot 2016-06-14 at 12 14 33 am

3. Changes disclaimer color to neutral, and text color to `$base` to match (new icons not shown)
screen shot 2016-06-13 at 12 07 21 pm

4. Small changeout on the "go" button icon, from `arrow-right-border`, to just `arrow-right` to keep it simple

## Review:
@noahmanger (& a heads up to @adborden)
- Dependent PR: https://github.com/18F/fec-cms/pull/340

### Notes:
- Would love 👀 on the icon implementations specifically. By the end of this, my style guide and cms local instances weren't talking to each other anymore, or updating. I'm 100% certain that the sizing in `_cards.scss` will need to be adjusted, I just plunked them in.

### CODE REVIEW

File: scss/components/_cards.scss [hunk 1]
```diff
@@ -271,3 +271,21 @@
     }
   }
 }
+
+.card--secondary {
```
Comment (jenniferthibault) [inline scss/components/_cards.scss]: I'm 100% positive the sizes for these will need to be adjusted

**Comment by jenniferthibault** [comment]
I should also note that the reason I made all these changes is to get ahead of the changes needed in https://github.com/18F/fec-cms/issues/298 : cross-linking calendar reporting deadlines from other parts of the Registration/reporting page

**Comment by adborden** [comment]
:+1:

File: scss/components/_cards.scss [hunk 2]
```diff
@@ -271,3 +271,21 @@
...
+  .card__icon {
+    &.i-calendar {
+      @include u-icon-bg($calendar, $secondary-contrast);
```
Comment (adborden) [inline scss/components/_cards.scss]: You deleted the calendar declaration above but are still using it here. That's why Travis is complaining https://travis-ci.org/18F/fec-style/builds/137429094#L1066

**Pull request merged**